src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--start-day--sun calendar--month-length--29">
```

Comment:
```suggestion
    <main class="calendar calendar--sun calendar--29">
```
Классы можно вот так называть